Source/ACE.DatLoader/DatManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Linq;
+using System.IO;
+
+using ACE.Common;
+
+namespace ACE.DatLoader
+{
+    public class DatManager
+    {
+        public static DatDatabase celldat;
```

Comment:
public properties should be Proper Case - stylecop should be complaining about this (ie, CellDat and PortalDat).  Also, this is a bad use of a static member because it can get into an invalid state (ie, before Initialize is called).  This shouldn't happen in practice, I understand, but it's bad class design.

If you want to keep the static properties, that's fine, but Initialize logic should be moved to a static constructor if so.